src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java

```diff
@@ -107,4 +108,30 @@ private void uploadFile(MultipartFile file, UUID id) {
         return formFullDTOList;
 
     }
+
+    public void restoreFormById(UUID formId) {
+        putClassNameInMDC();
+        log.info("Try to activate form with uuid= " + formId);
```

Comment:
Please, synchronize with team and try to follow the same style.
In PR #28  @Maksim-maevsky starts using of [substitution approach for logging](https://github.com/viktarkolbik/backend_team_four/pull/28#discussion_r614719966).

Reply:
Done

Reply:
Thank you!